package.json

```diff
@@ -18,6 +18,7 @@
     "url": "git+ssh://git@github.com/Expensify/JS-Libs.git"
   },
   "dependencies": {
+    "@octokit/rest": "^18.3.1",
```

Comment:
Curious if this needs to be a production dependency. Since the `octokit` client will be passed in from other places in code, and is just referenced here. I think the only place we're actually going to need to access the module itself is in tests, so maybe this could be a dev dependency.

Reply:
Good call! Let me try to make it a dev dependency.